src/otx/core/data/entity/visual_prompting.py

```diff
@@ -127,26 +130,16 @@ class VisualPromptingBatchPredEntity(OTXBatchPredEntity, VisualPromptingBatchDat
     """Data entity to represent model output predictions for visual prompting task."""
 
 
-@dataclass
-class ZeroShotVisualPromptingLabel:
```

Comment:
It's just a question. why is it unnecessary now?

Reply:
Before, I tried to manage prompts as different key-value pairs and through this update prompts are combined into a list of prompts. As a result, this data class is unnecessary anymore.